app/models/service_template.rb

```diff
@@ -356,4 +361,19 @@ def create_resource_actions(ae_endpoints)
     end
     save!
   end
+
+  private
+
+  def construct_config_info
+    config_info = {}
+    if service_resources.where(:resource_type => 'MiqRequest').exists?
+      config_info.merge!(service_resources.find_by(:resource_type => 'MiqRequest').resource.options)
+    end
+
+    resource_actions.each do |resource_action|
+      resource_options = resource_action.attributes.merge!(:fqname => resource_action.fqname)
```

Comment:
we don't want everything, only `dialog_id`, `fqname`, `configuration_template_id`, `configuration_template_type`. Also include them when not `nil`